specification/edgeorderpartner/resource-manager/Microsoft.EdgeOrderPartner/preview/2020-12-01-preview/edgeorderpartner.json

```diff
@@ -0,0 +1,905 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "searchInventoriesRequest",
+            "in": "body",
+            "description": "Searches inventories with the given filters and returns in the form of a list",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SearchInventoriesRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/PartnerInventoryList"
+            }
+          },
+          "default": {
+            "description": "Error response describing reason for operation failure.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "SearchInventories": {
+            "$ref": "./examples/SearchInventories.json"
+          },
+          "SearchInventoriesDetails": {
+            "$ref": "./examples/SearchInventoriesDetails.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AdditionalErrorInfo": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string"
+        },
+        "info": {
+          "type": "object"
+        }
+      }
+    },
+    "AdditionalInventoryDetails": {
+      "description": "Contains additional data about inventory in dictionary format",
+      "type": "object",
+      "properties": {
+        "additionalData": {
+          "description": "Additional Data",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "AdditionalOrderItemDetails": {
+      "description": "Contains additional order item details",
+      "type": "object",
+      "properties": {
+        "status": {
+          "$ref": "#/definitions/StageDetails",
+          "description": "Order item status",
+          "readOnly": true
+        },
+        "subscription": {
+          "$ref": "#/definitions/SubscriptionDetails",
+          "description": "Subscription details",
+          "readOnly": true
+        }
+      }
+    },
+    "BillingDetails": {
+      "description": "Contains billing details for the inventory",
+      "type": "object",
+      "properties": {
+        "billingType": {
+          "description": "Billing type for the inventory",
+          "type": "string",
+          "readOnly": true
+        },
+        "status": {
+          "description": "Billing status for the inventory",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "CloudError": {
```

Comment:
Why have `CloudError` when no one uses it? Please try clean up the unused schemas.

Reply:
We will take all these comments in next PR.